spec/SPEC.md

```diff
@@ -0,0 +1,301 @@
+# OmniBOR Specification
+
+| Field   | Value |
+|:--------|:------|
+| Version | 0.1   |
+| Status  | Draft |
+
+## Foreword
+
+This specification is subject to the Community Specification License 1.0,
+available at <https://github.com/omnibor/spec>.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" are used as described in
+[RFC 2119][rfc_2119].
+
+Attention is drawn to the possibility that some of the elements of this
+document may be the subject of patent rights. No party shall be held
+responsible for identifying any or all such patent rights.
+
+Any trade name used in this document is information given for the convenience
+of users and does not constitute an endorsement.
+
+This document was prepared by the OmniBOR Community.
+
+Known patent licensing exclusions are available in the specification
+repository's `NOTICES.md` file.
+
+Any feedback or questions on this document should be directed to the
+specification's repository, located at <https://github.com/omnibor/spec>.
+
+THESE MATERIALS ARE PROVIDED "AS IS." The Contributors and Licensees expressly
+disclaim any warranties (express, implied, or otherwise), including implied
+warranties of merchantability, non-infringement, fitness for a particular
+purpose, or title, related to the materials. The entire risk as to
+implementing or otherwise using the materials is assumed by the implementer and
+user. IN NO EVENT WILL THE CONTRIBUTORS OR LICENSEES BE LIABLE TO ANY OTHER
+PARTY FOR LOST PROFITS OR ANY FORM OF INDIRECT, SPECIAL, INCIDENTAL, OR
+CONSEQUENTIAL DAMAGES OF ANY CHARACTER FROM ANY CAUSES OF ACTION OF ANY KIND
+WITH RESPECT TO THIS DELIVERABLE OR ITS GOVERNING AGREEMENT, WHETHER BASED ON
+BREACH OF CONTRACT, TORT (INCLUDING NEGLIGENCE), OR OTHERWISE, AND WHETHER OR
+NOT THE OTHER MEMBER HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+## Introduction
+
+Software supply chains face many challenges: security and compliance chief
+among them. Often, projects are hamstrung by the inability to easily and
+reliably capture a complete, concise, verifiable accounting of exactly
+__what__ inputs were built into software. Without this information,
+identifying vulnerable software to patch or replace is difficult. While
+Software Bills of Material (SBOMs) help identify third-party components,
+they do not go far enough to precisely identify the exact inputs necessary
+for vulnerability management.
+
+The OmniBOR standard defines three concepts, which together enable the
+consistent, reproducible, and embeddable encoding of the exact inputs used to
+build a software artifact: Artifact Identifiers, Input Manifests, and Artifact
```

Comment:
I would still favor 'Artifact Input Manifests' rather than 'Input Manifest'.

Reply:
There is a symmetry to each of the Three Concepts beginning with the word "Artifact" which I find appealing.